recipes/AISHELL-1/ASR/CTC/prepare.py

```diff
@@ -0,0 +1 @@
+../../prepare.py
```

Comment:
Hey, the naming convention of the file is not equivalent to other recipes, please have a look at the name and just rename the file accordingly.